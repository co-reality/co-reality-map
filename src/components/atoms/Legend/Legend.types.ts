export interface LegendProps {
  text: string;
  position?: "left" | "right";
  onClick?: () => void;
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  icon?: any;
}
