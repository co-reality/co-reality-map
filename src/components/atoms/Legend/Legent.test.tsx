import React from "react";
import renderer from "react-test-renderer";

// Test component
import Legend from "./Legend";

describe("<Legend />", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Legend text="Legent Test" />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
