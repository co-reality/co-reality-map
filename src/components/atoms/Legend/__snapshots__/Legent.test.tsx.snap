// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Legend /> renders correctly 1`] = `
<div
  className="sc-bdfBwQ Krtgc"
>
  Legent Test
</div>
`;
